src/components/HeroPanel/AyaHeroPanel.js

```diff
@@ -38,12 +38,11 @@ export default class HeroPanel extends Component {
             </div>
             <div className="hero-panel-type">
               <div className="hero-panel-type-lining">
-                <div className={type}>
+                <div className={type + ' siteHeadingAya'}>
```

Comment:
A better way is es6 template string {`${type} siteHeadingAya`}

Can  siteHeadingAya be a local style